### PR TITLE
actions: update permissions and add show-vulnerabilities to security

### PR DESCRIPTION
**Pull request by iancardosozup**
Signed-off-by: Ian Cardoso <ian.cardoso@zup.com.br>

**- What I did**
Updating permissions to match [scorecard](https://github.com/ossf/scorecard) standards, but token permissions is still in [WIP](https://github.com/ossf/scorecard/issues/942) and it's a problemactic subject on scorecard yet. I will checkout updates to keep us informed about it.

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Review by matheusalcantarazup — CHANGES_REQUESTED** [review]
Please rebase with main to resolve conflicts.

**Review by nathanmartinszup — APPROVED** [review]

**Comment by matheusalcantarazup** [comment]
This PR contains commits that already been merged on main branch of Horusec. Please rebase your PR with our current main branch.

**Comment by iancardosozup** [comment]
> This PR contains commits that already been merged on main branch of Horusec. Please **rebase** your PR with our current main branch.

This PR depends on a feature i will add, as soon it's ok i'll update the branch

File: .github/workflows/security.yml [hunk 1]
```diff
@@ -28,4 +27,5 @@ jobs:
           HORUSEC_CLI_REPOSITORY_NAME: ${{ secrets.HORUSEC_CLI_REPOSITORY_NAME }}
         run: |
           curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/master/deployments/scripts/install.sh | bash -s latest
-          horusec start -p . -e true -G true
+          horusec start -p . -e true -G true --show-vulnerabilities-types="Vulnerability, Risk Accepted"
```
Comment (matheusalcantarazup) [inline .github/workflows/security.yml:30]: What do you think about remove the "true" values? They are not necessary when using boolean flags.

Comment (iancardosozup) [reply .github/workflows/security.yml:30]: please refer to #637 since we found a bug on PR with fork origin

**Pull request closed without merging**